pallets/runtime/src/bridge.rs

```diff
@@ -135,52 +176,60 @@ decl_module! {
 
         fn deposit_event() = default;
 
-        /// Change the signer set account as root.
+        /// Change the signer set account as root or as admin.
         pub fn change_relayers(origin, account_id: T::AccountId) -> DispatchResult {
-            ensure_root(origin)?;
+            Self::is_root_or_admin(origin)?;
```

Comment:
Can we change this to just admin. MIPs all execute as `root` and in this case we don't want to allow this function to be called through governance (only by the custodial Polymath admin key).